tools/workspace-plugin/src/executors/build/lib/babel.ts

```diff
@@ -25,7 +25,7 @@ export function hasStylesFilesToProcess(normalizedOptions: NormalizedOptions) {
   return files.length > 0;
 }
 
-export async function compileWithGriffelStylesAOT(options: NormalizedOptions, successCallback: () => Promise<boolean>) {
+export async function compileWithGriffelStylesAOT(options: NormalizedOptions) {
```

Comment:
that successCallback param drilling was never a good idea - now it's gonn as invocation is composod within executor main funciton